src/main/webapp/app/lecture/manage/lecture-units/attachment-video-units/attachment-video-units.component.spec.ts

```
@@ -0,0 +1,283 @@
+        } as any);
+
+        attachmentVideoUnitsComponentFixture = TestBed.createComponent(AttachmentVideoUnitsComponent);
+        attachmentVideoUnitsComponent = attachmentVideoUnitsComponentFixture.componentInstance;
+        attachmentVideoUnitsComponentFixture.detectChanges();
+
+        attachmentVideoUnitsComponent.units = units;
+        attachmentVideoUnitsComponent.numberOfPages = numberOfPages;
+
+        attachmentVideoUnitService = TestBed.inject(AttachmentVideoUnitService);
+        router = TestBed.get(Router);
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)